sipn_reanalysis_ingest/luigitasks/untar.py

```diff
@@ -57,6 +59,30 @@ def output(self):
         )
 
 
+class UntarCfsr1DayFile(UntarFileTask):
+    """Untar a daily CFSR tar file."""
+
+    window_start = luigi.DateParameter()
+    window_end = luigi.DateParameter()
```

Comment:
Based on a suggestion above, I think we should switch to accepting a single `date` parameter here instead of a window.